src/services/ListService.ts

```diff
@@ -0,0 +1,65 @@
+export const setListSession = (sorting: string) => {
```

Comment:
@mdshamoon Can you please add documentation for each method here. What they do etc.